src/app/functions/server/sessionHelpers.ts

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { cookies } from "next/headers";
+import { NextResponse } from "next/server";
+import { getIronSession, IronSession, unsealData } from "iron-session";
+import { UserData } from "../../transitionTypes";
+
+export type SessionData = Partial<{
+  oauthState: string;
+  user: UserData;
+}>;
+
+const sessionConfig = {
+  cookieName: "monitor-iron-session",
+  password: process.env.IRON_SESSION_PASSPHRASE!,
+  cookieOptions: {
+    secure: process.env.NODE_ENV !== "development",
```

Comment:
Not sure if we need some helper to determine whether we're in a specific environment. I noticed we seem to do that quite often. Where we're checking if we're in development, NOT development, heroku, NOT production, or one suspicious check against "dev" (not "development").

```sh
git grep -En "process.env.NODE_ENV \!?={2,3}" | cat

next.config.js:19:              `script-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-eval' 'unsafe-inline'" : ''} https://*.googletagmanager.com`,
next.config.js:21:              `connect-src 'self' ${process.env.NODE_ENV === 'development' ? 'webpack://*' : ''} https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com`,
next.config.js:24:              `style-src 'self' ${process.env.NODE_ENV === 'development' ? "'unsafe-inline'" : ''}`,
src/app/functions/server/sessionHelpers.ts:19:    secure: process.env.NODE_ENV !== "development",
src/appConstants.js:60:if (!process.env.SERVER_URL && process.env.NODE_ENV === 'heroku') {
src/middleware/error.js:41:    message: process.env.NODE_ENV !== 'production' ? errMsg : 'Something went wrong', // hide error message when in production
src/middleware/error.js:42:    stack: process.env.NODE_ENV === 'dev' ? err.stack : {} // hide stack when not in dev
```

Reply:
Yeah, that `dev` one seems suspicious. I’ll take a look!

Reply:
@Vinnl Seems like we would need to change `process.env.NODE_ENV === 'development'` to `process.env.NODE_ENV === 'dev'` in `next.config.js`?